gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
This file along with the other 4 files constitute all of the gradle distribution that needs to be in this project.  On first run, this will detect that gradle is not installed and download the appropriate files.  Thereafter, everything will run locally.  Very slick!